README.md

```diff
@@ -1,2 +1,55 @@
 # SmartVote
+
 A voting software, decentralized and secured running on the blockchain.
+
+## Introduction
+
+The idea of SmartVote (SmVo) is to make it possible to follow and participate to an election without having to be vigilant
```

Comment:
Ask for required changes about the project description to your projects leaders.